packages/bigtable/src/table.js

```diff
@@ -175,26 +154,36 @@ function Table(bigtable, name) {
 util.inherits(Table, common.GrpcServiceObject);
 
 /**
+ * @private
```

Comment:
Sure, I copied the description from the proto files.. hopefully that will suffice?